### ISSUE TITLE
Take the control over this module and merge the best

### ISSUE THREAD

**Issue by NicolasDumoulin** [Alem/django-jfu#44]
Hi all,

The maintener of this module seems lost from here, and a lot of forks have been done for fixing a lot of various issues.
The fact is that the module on pypi seems to refer to this outdated repo.

Is there a way to select a fork as a new reference in which pull issue will be merged, or several contributors will be integrated to the project?
Then, we may try to merge all the forks.

Thanks

**Comment by timonweb** [comment]
I second it, looks like the module has been abandoned

**Comment by ghost** [comment]
Yep ! more or less theres no one who's ready to maintain this probably ! may be a mail can be a help ?
